common/src/lib.rs

```diff
@@ -38,9 +38,9 @@ pub use fuse::{FuseLogEntry, FuseLogEntryId};
 pub use pcr::{PcrLogEntry, PcrLogEntryId, RT_FW_CURRENT_PCR, RT_FW_JOURNEY_PCR};
 
 pub const FMC_ORG: u32 = 0x40000000;
-pub const FMC_SIZE: u32 = 20 * 1024;
+pub const FMC_SIZE: u32 = 22 * 1024;
 pub const RUNTIME_ORG: u32 = FMC_ORG + FMC_SIZE;
-pub const RUNTIME_SIZE: u32 = 92 * 1024;
+pub const RUNTIME_SIZE: u32 = 90 * 1024;
```

Comment:
You can fix the overflow by increasing the size of the binary. We got 128KB of ICCM.
pub const RUNTIME_SIZE: u32 = 92 * 1024;

Reply:
Thanks! I had been keeping the total size constant between FMC and RT FW, but guess that wasn't necessary.